### ISSUE TITLE
[Question] What is difference between darknet on this github and on vcpkg?

### ISSUE THREAD

**Issue by yes89929** [AlexeyAB/darknet#6868]
Hi.
I`m new on darknet.

I installed darknet following manual on this github.
```
PS Code\>              git clone https://github.com/microsoft/vcpkg
PS Code\>              cd vcpkg
PS Code\vcpkg>         $env:VCPKG_ROOT=$PWD
PS Code\vcpkg>         .\bootstrap-vcpkg.bat
PS Code\vcpkg>         .\vcpkg install darknet[full]:x64-windows #replace with darknet[opencv-base,cuda,cudnn]:x64-windows for a quicker install of dependencies
PS Code\vcpkg>         cd ..
PS Code\>              git clone https://github.com/AlexeyAB/darknet
PS Code\>              cd darknet
PS Code\darknet>       .\build.ps1
```

When I first looked at the manual, I thought that vcpkg's darknet[full] package is to install darknet's 3rd party library.
However, vcpkg's darknet[full] also provides the darknet library.

Can I train and test YOLO v4 model only using vcpkg`s darknet?

**Comment by stephanecharette** [comment]
> Can I train and test YOLO v4 model only using vcpkg`s darknet?

Yes, you can use it to train and test models.  There shouldn't be any (significant?) difference between building it yourself and having vcpkg build it.

**Comment by stephanecharette** [comment]
If I remember correctly, "full" also downloads some default pre-built weights files, it doesn't just build darknet.

**Comment by yes89929** [comment]
> If I remember correctly, "full" also downloads some default pre-built weights files, it doesn't just build darknet.

Thank you for reply

Yes. training data, cfg, weights are in vcpkg/packages>darkent_x64-windows/tool/darknet.

So, I understood as below.
The installation manual on this github use vcpkg darknet[full] for installation of 3rd party libraries.
And, difference bewteen this github and vcpkg is just version and some files.
Right?
